templates/header.php

```diff
@@ -6,8 +6,23 @@
   <div class="header__inner">
     <?php get_template_part('patterns/components/navigation-toggle'); ?>
     <div class="header__unify-logo-nav">
-      <a href="<?= esc_url(home_url('/')); ?>" class="logo__link logo__link--horiz theme--primary-background-color show-until--large"><img src="<?php bloginfo('template_directory'); ?>/dist/images/sda-logo--horiz.svg" alt="<?php bloginfo('name'); ?> - logo"></a>
-      <a href="/" class="logo__link logo__link--square theme--primary-background-color show-at--large"><img src="<?php bloginfo('template_directory'); ?>/dist/images/sda-logo--square.svg" alt="<?php bloginfo('name'); ?> - logo"></a>
+      <?php
+        $logo_square = get_field('logo_square', 'options');
+        $logo_horizontal = get_field('logo_horizontal', 'options');
+        $logo_text = get_field('logo_text', 'options');
+      ?>
+      <?php if ($logo_horizontal): ?>
+        <a href="<?= esc_url(home_url('/')); ?>" class="logo__link logo__link--horiz theme--primary-background-color show-until--large <?php if ($logo_text): echo 'logo--with-text'; endif; ?>">
```

Comment:
@kelscahill I know this was probably in here already, but let's update this shorthand `php` to actually `echo` that url.